plugins/modules/ndo_port_channel_interface.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: ndo_port_channel_interface
+short_description: Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.2 (NDO v4.4) and later.
+author:
+- Gaspard Micol (@gmicol)
+options:
+    template:
+        description:
+        - The name of the template.
+        - The template must be a tenant template.
+        type: str
+        required: true
+    port_channel_interface:
+        description:
+        - The name of the Port Channel Interface.
+        type: str
+        aliases: [ name, port_channel ]
+    port_channel_interface_uuid:
+        description:
+        - The uuid of the Port Channel Interface.
+        - This parameter is required when the O(port_channel_interface) needs to be updated.
+        type: str
+        aliases: [ uuid, port_channel_uuid ]
+    description:
+        description:
+        - The description of the Port Channel Interface.
+        type: str
+    node:
+        description:
+        - The node ID.
+        type: str
+    member_interfaces:
```

Comment:
Should we allow range input as in ui? 

Or should we add an example that leverages the aci ranges lookup plugin: https://docs.ansible.com/ansible/latest/collections/cisco/aci/interface_range_lookup.html#ansible-collections-cisco-aci-interface-range-lookup?

Reply:
we already accepts range inputs, I tested and it worked fine. I decided to go with a list to replace the commas between the different inputs. so instead of having to put `"1/1,1/3-5,1/18"` for example, it would be like:

```YML
member_interfaces:
  - 1/1
  - 1/3-5
  - 1/18
```

I don't know if it's the best way to do it. let me know what you think.

Reply:
yes, will add it as well